.travis.yml

```diff
@@ -45,10 +45,12 @@ matrix:
     - python: "3.6"
       before_script:
         - python3.6 -m pip install black==18.9b0
-        - pip install -r source/jormungandr/requirements_dev.txt
+        - pip install -r requirements_pre-commit.txt
+        - pip install pre-commit
```

Comment:
Woo and it currently works ? Thus, pre-commit is installed by default on `Travis` I guess